consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/BaseBftProtocolSchedule.java

```diff
@@ -82,21 +76,18 @@ public ProtocolSchedule createProtocolSchedule(
       final BftConfigOptions config, final FeeMarket feeMarket);
 
   private ProtocolSpecBuilder applyBftChanges(
-      final BftConfigOptions configOptions,
       final ProtocolSpecBuilder builder,
+      final BftConfigOptions configOptions,
       final boolean goQuorumMode,
-      final BftExtraDataCodec bftExtraDataCodec,
-      final Optional<BigInteger> blockReward) {
```

Comment:
Simplify method signature: just pull the `blockReward` from `configOptions` which is already provided